src/common/headlessService/utils.ts

```diff
@@ -106,14 +109,25 @@ export function isEventSelectedCarousel(
   );
 }
 
+export function isLocationsSelected(
+  module: PageModule | PageSidebarModule,
+): module is LocationsSelected {
+  return (
+    (<LocationsSelected>module).locations !== undefined &&
+    // eslint-disable-next-line no-underscore-dangle
+    module.__typename === 'LocationsSelected'
+  );
+}
+
```

Comment:
Shouldn't there be one for the location carousel as well? It should also be used in some function like `isVenueModule`.